api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -116,9 +116,8 @@
   public ResponseEntity<CriteriaListResponse> getCriteriaByTypeForCodeOrName(Long cdrVersionId,
                                                                              String type,
                                                                              String value) {
-    //TODO: really implement this
     cdrVersionService.setCdrVersion(cdrVersionDao.findOne(cdrVersionId));
-    final List<Criteria> criteriaList = criteriaDao.findCriteriaByType("demo");
+    final List<Criteria> criteriaList = criteriaDao.findCriteriaByTypeForCodeOrName(type, value);
```

Comment:
this is used by any or all types? what is value?

Reply:
Yes, this can be used for any tree per say. It represents a search value.

Reply:
oh right, for criteria search. ok.